<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieMakerApp</class>
 <widget class="QMainWindow" name="MovieMakerApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1085</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MovieMakerApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="fcDataComboBox"/>
      </item>
      <item>
       <widget class="QTableWidget" name="channelTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>use</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>idx</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>label</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="movieFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="movieFrameLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="movieControlsLayout">
        <item>
         <widget class="QSlider" name="movieFrameSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="movieFrameLabel">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Data_Folder"/>
    <addaction name="actionLoad_FC_File"/>
    <addaction name="actionLoad_node_layout"/>
    <addaction name="actionLoad_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionExport_movie"/>
    <addaction name="actionSave_node_layout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAutogen_Circles"/>
    <addaction name="separator"/>
    <addaction name="actionLabel_Nodes"/>
    <addaction name="actionActivity_Based_Nodes"/>
    <addaction name="actionUndirected_Edges"/>
    <addaction name="actionFC_Based_Edges"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Data_Folder">
   <property name="text">
    <string>Load data folder...</string>
   </property>
  </action>
  <action name="actionLoad_FC_File">
   <property name="text">
    <string>Load FC file...</string>
   </property>
  </action>
  <action name="actionLoad_node_layout">
   <property name="text">
    <string>Load node layout...</string>
   </property>
  </action>
  <action name="actionExport_movie">
   <property name="text">
    <string>Export movie...</string>
   </property>
  </action>
  <action name="actionSave_node_layout">
   <property name="text">
    <string>Save node layout...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit...</string>
   </property>
  </action>
  <action name="actionLoad_Recent">
   <property name="text">
    <string>Load Recent</string>
   </property>
  </action>
  <action name="actionUpdate_Plot">
   <property name="text">
    <string>Update Plot</string>
   </property>
  </action>
  <action name="actionAutogen_Circles">
   <property name="text">
    <string>Autogen Circles</string>
   </property>
  </action>
  <action name="actionLabel_Nodes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Label Nodes</string>
   </property>
  </action>
  <action name="actionActivity_Based_Nodes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Activity-Based Nodes</string>
   </property>
  </action>
  <action name="actionUndirected_Edges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Undirected Edges</string>
   </property>
  </action>
  <action name="actionFC_Based_Edges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FC-Based Edges</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
